src/content/docs/tutorial-customer-experience/improve-customer-experience.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: "Improve your customer experience" 
+metaDescription: "New Relic observability maturity series: this guide gives you a quality foundation for optimizing your web application's performance to improve your customer experience."
+redirects:
+---
+
+After you've [established your current state](/docs/tutorial-customer-experience/establish-current-state/), it's time to take the information you've gathered and use it to start improving your customer experience. Since needs can vary based on each organization, you may have different areas you need to improve. The following steps are a suggested order to use for improving your customers' experience, and you may find that you can skip some steps based on your KPIs. As this is a continuing process, we recommend you bookmark this page so that you can refer to it in the future as your KPIs change. 
+
+<Steps>
+    <Step>
+### Decide which KPIs to improve
+
+After you [establish your current state](/docs/tutorial-customer-experience/establish-current-state/), you know what your user experience looks like across your system. From there, you need to determine which of the following areas need to improve:
+
+1. **Start with business priorities:** If you have clear business directives, you should focus on what matters most to your organization. For example, if your company has recently launched a new initiative around a line of business but the KPIs associated with the UI are below target, this is a great place to focus your efforts.
+2. **Focus on your KPIs:** Your key performance indicators are called "key" for a reason! They provide hard data on the various aspects of your system that act as pain points for your customers.
+3. **Filter each segment:** Take each of your segments that you created in the previous doc and group them to see if you need to focus more on specific regions, devices, or any other helpful grouping.        
+    </Step>
+    <Step>
+### Improve targeted KPIs
+
+To track your progress, create a new dashboard (or add a new page to the existing dashboard) and name it `Quality Foundation KPI Improvement`. For more information, see [Improve web uptime](/docs/new-relic-solutions/observability-maturity/customer-experience/cx-improve-web-uptime).        
+    </Step>
+    <Step>
+### Improve page load performance
+
+Once you've created the dashboard, it's time to narrow your focus to the specific pages that aren't meeting target KPI values. 
```

Comment:
The "Quality foundation" dashboard?

What's tricky about this doc is that in the intro you tell people to pick and choose, but these early sections seem pretty linear/interrelated. 

Maybe there's a way you can group info in this doc in "required" and "optional but useful" sections? (not those literal category names, but I think you get the idea)

Reply:
Maybe the correct thing to do here is to remove the steps component and just list these as things you can do? We'll try that and see if we get feedback on wanting a more linear experience. Thanks for the suggestion!

Reply:
@jeff-colucci Yup, let's keep it for now and see how it goes :)